pykeepass/entry.py

```diff
@@ -294,6 +294,25 @@ def delete_custom_property(self, key):
             raise AttributeError('Could not find property element')
         self._element.remove(prop)
 
+    def is_custom_property_protected(self, key):
+        """Whether the attribute with a specific key is protected.
+
+        If the key does not exists it returns False.
+
+        Returns:
+            bool: Whether the attribute is protected.
+        """
+        assert key not in reserved_keys, '{} is a reserved key'.format(key)
+        field = self._xpath(
+            'String/Key[text()="{}"]/../Value'.format(key), first=True
+        )
+        if field is not None:
+            for key_, value in zip(field.keys(), field.values()):
+                if key_ == "Protected" and value == "True":
+                    return True
+
+        return False
```

Comment:
Should be able to just do `return bool(field.attrib.get("Protected", False))

Reply:
Thanks, note though that `bool("False")` actually returns `True` so one has to compare it with the string `"True"`.